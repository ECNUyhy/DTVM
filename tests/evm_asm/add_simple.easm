// 1 + 2
PUSH1 0x02
PUSH1 0x01
ADD
