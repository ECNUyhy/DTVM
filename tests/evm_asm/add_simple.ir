const.i64 1
const.i64 0
const.i64 0
const.i64 0
const.i64 2
const.i64 0
const.i64 0
const.i64 0
const.i64 0
add (const.i64 1, const.i64 2)
adc (const.i64 0, const.i64 0, const.i64 0)
adc (const.i64 0, const.i64 0, const.i64 0)
adc (const.i64 0, const.i64 0, const.i64 0)
