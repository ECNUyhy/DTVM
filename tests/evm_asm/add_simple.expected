status: success
error_code: 0
stack: [3]
memory: ''
storage: {}
transient_storage: {}
return: ''
events: []
